package.json

```diff
@@ -139,7 +139,7 @@
 		"@rocket.chat/ui-kit": "^0.7.1",
 		"@slack/client": "^4.8.0",
 		"adm-zip": "RocketChat/adm-zip",
-		"apn": "2.2.0",
+		"apn": "^2.2.0",
```

Comment:
Is this update really needed?

Reply:
i think its has updated automatically but i have changed as previous